docs/minio-admin-complete-guide.md

```diff
@@ -54,17 +54,17 @@ mc.exe --help
 ```
 
 ### Install from Source
-Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://min.io/downloads/#minio-client.
```

Comment:
Same URL issue, change `downloads` to `download`

Reply:
Done